provider/schema_spl_dashboard_chart.go

```diff
@@ -61,14 +61,67 @@ func schemaDashboardChart() map[string]*schema.Schema {
 						Type:     schema.TypeString,
 						Required: true,
 					},
+					"label": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default: nil,
+					},
+					"collection": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default: "default",
+					},
+					"hidden": {
+						Type:     schema.TypeBool,
+						Optional: true,
+						Default: nil,
+					},
+					"query_group_unit": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default: "",
+					},
+					"query_group_function": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default: "",
+					},
 					"expression_plain": {
 						Type:     schema.TypeString,
 						Required: true,
 					},
+					"query_filter_asset_id": {
+						Type:     schema.TypeString,
+						Required: true,
+					},
+					"query_filter_asset_name": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default: nil,
+					},
+					"query_filter_attribute_id": {
+						Type:     schema.TypeString,
+						Required: true,
+					},
+					"query_filter_attribute_name": {
+						Type:     schema.TypeString,
+						Optional: true,
```

Comment:
nested maybe?

Reply:
I tried to use that but I don't know how to do it, if you know it let do a short call